backend/models/todoList.model.js

```diff
@@ -1,25 +1,25 @@
-const mongoose = require('mongoose')
+const mongoose = require("mongoose");
 const todoSchema = new mongoose.Schema({
```

Comment:
what about user relationship i mean who has added which todo how we will know this.